test/config/chaosduck/chaosduck.yaml

```diff
@@ -80,6 +80,8 @@ spec:
         args: [
           # TODO(https://github.com/knative/eventing/issues/3590): Enable once IMC chaos issues are fixed.
           "-disable=imc-controller", "-disable=imc-dispatcher",
+          # Disable chaos on webhooks until https://github.com/knative/pkg/issues/1509 is sorted out.
+          "-disable=eventing-webhook",
```

Comment:
nit: Can we add https://github.com/knative/pkg/issues/1509 to the list of issues above to solve to enable chaos on IMC?

Reply:
Not sure I follow, you mean you want me to comment that this issue affects both imc-controller and the eventing-webhook?

Reply:
Yes, by reading reading your comment at line 83, it's unclear that the issue affects imc-controller that is disabled at line 82.

Reply:
Ah, I didn't know it did